src/app/views/settings/Settings.tsx

```diff
@@ -113,6 +122,10 @@ function Settings(props: ISettingsProps) {
       });
   };
 
+  const handleChangeMode = (permissionModeType: boolean) => {
+    dispatch(changeMode(!permissionModeType));
+  };
+
```

Comment:
do you think this function should be moved to the `src/app/services/actions/permission-mode-action-creator.ts` like in `src/app/services/actions/theme-action-creator.ts`? i initally copied this file: `src/app/services/actions/toggle-sidebar-action-creator.ts`.